### PR TITLE
Benine view preparations

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Timeline-Toast element now per Timeline-Block.
- Toast updated with cleaned layout.
- Added dialog for presets.
- Added non-functional button to drawer for benine.

### CODE REVIEW

**Comment by bart-de-jonge** [comment]
Nice work extracting the toast to the shots. I don't see weird stuff, looks good to me

File: elements/gotofail-drawer.html [hunk 1]
```diff
@@ -104,28 +110,30 @@
...
+                <a href="/director" tabindex="-1" style="text-decoration:none"> <paper-item class="item-style separator-top separator-bottom">
                     <paper-ripple></paper-ripple>
                     <iron-icon class="icon-style" icon="image:camera-alt"></iron-icon>
                     <span class="text-style">Director view</span>
                 </paper-item> </a>
 
+                <a tabindex="-1" style="text-decoration:none"> <paper-item class="item-style separator-top">
+                    <paper-ripple></paper-ripple>
+                    <iron-icon class="icon-style" icon="image:linked-camera"></iron-icon>
+                    <span class="text-style">Be-Nine application</span>
```
Comment (martijnjanssen) [inline elements/gotofail-drawer.html]: It is spelled 'BeNine'.

**Comment by martijnjanssen** [comment]
Looks good!

File: elements/gotofail-drawer.html (hunk 1 above)
Comment (Markvanderuit) [inline elements/gotofail-drawer.html]: Derp. Fixed.

**Pull request merged**